tests/Testcases/trivial/test.tcl

```diff
@@ -1,4 +1,5 @@
 create_design test
+architecture ../../Arch/k6_frac_N10_tileable_40nm.xml ../../Arch/k6_N10_40nm_openfpga.xml
```

Comment:
Can we add a negative test in one of the top-level cases ?

Reply:
No, we can't. let's discuss this case later.

Reply:
Dummy compiler

Reply:
Test will be added at Raptor level.